<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inscrit</class>
 <widget class="QDialog" name="inscrit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>707</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>623</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>623</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

*{
	font-family:century gothic;
	font-size:24px;
}
QDialog{
background-color: rgb(39, 44, 54);
}
QFrame
{
	background:rgb(39, 44, 54);
}

QPushButton
{
background-color: red ;/** for example **/
border-radius:10px;

}
QToolButton
{
image:url(:img/login.png);
background-color: red ;/** for example **/
border-radius:60px;

}
QLabel
{
color:white;
}
QMainWindow
{
background:transparent;
}

QPushButton:hover
{
background-color:rgb(170, 0, 0)  ;/** for example **/
	
border-radius:5px;

}
QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>INSCRIPTION</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/login.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>580</y>
        <width>561</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>S'inscrir</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>410</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>500</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QCheckBox::indicator:checked {
	background-color: rgb(170, 0, 0);

}
QCheckBox {
background-color: rgb(85, 170, 255);
color :rgb(255, 255, 255);
}
QCheckBox:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QCheckBox:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}
</string>
      </property>
      <property name="text">
       <string>GÃ©rent</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>500</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QCheckBox::indicator:checked {
	background-color: rgb(170, 0, 0);

}
QCheckBox {
background-color: rgb(85, 170, 255);
color :rgb(255, 255, 255);
}
QCheckBox:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QCheckBox:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}
</string>
      </property>
      <property name="text">
       <string>Caissier</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
